library/Icingadb/Widget/ItemList/RedundancyGroupListItem.php

```diff
@@ -0,0 +1,156 @@
+            ->with($relations)
+            ->filter($filter);
+
+        $members = RedundancyGroup::on($this->getDb())
+            ->columns([
+                'id' => 'id',
+                'objects_output' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.output IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.output'
+                    . ' ELSE redundancy_group_from_to_host_state.output END'
+                ),
+                'objects_long_output' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.long_output IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.long_output'
+                    . ' ELSE redundancy_group_from_to_host_state.long_output END'
+                ),
+                'objects_checkcommand_name' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host.checkcommand_name IS NULL'
+                    . ' THEN redundancy_group_from_to_service.checkcommand_name'
+                    . ' ELSE redundancy_group_from_to_host.checkcommand_name END'
+                ),
+                'objects_last_state_change' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.last_state_change IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.last_state_change'
+                    . ' ELSE redundancy_group_from_to_host_state.last_state_change END'
+                ),
+                'objects_severity' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.severity IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.severity'
+                    . ' ELSE redundancy_group_from_to_host_state.severity END'
+                )
+            ])
+            ->with($relations)
+            ->filter($filter)
+            ->orderBy([
+                'objects_severity',
+                'objects_last_state_change',
+            ], SORT_DESC);
+
+        $this->applyRestrictions($members);
+
+        /** @var RedundancyGroup $data */
+        $data = $members->first();
+
+        if($data) {
+            $caption->addHtml(new PluginOutputContainer(
+                (new PluginOutput($data->objects_output . "\n" .$data->objects_long_output))
+                    ->setCommandName($data->objects_checkcommand_name)
+            ));
+        }
+
+        $caption->addHtml(new ObjectsStatistics($summary->first()));
+    }
+
+    protected function assembleTitle(BaseHtmlElement $title): void
+    {
+        $subject = $this->createSubject();
+        if ($this->state->failed) {
+            $stateTextElement = Html::sprintf(
+                t('%s has %s', '<hostname> has <state-text>'),
+                $subject,
+                new HtmlElement('span', Attributes::create(['class' => 'state-text']), Text::create('FAILED'))
```

Comment:
As discussed offline with @nilmerg this will not be a state-text instead the text will be `has no working objects` if the state is failed else `has working objects`.